src/CMakeLists.txt

```diff
@@ -523,11 +526,11 @@ SET(OpenMVG_INCLUDE_DIRS
 
 INCLUDE_DIRECTORIES(${OpenMVG_INCLUDE_DIRS})
 
-find_package(Popsift)
-if(POPSIFT_FOUND)
+find_package(PopSift)
+if(PopSift_FOUND)
   ADD_DEFINITIONS(-DHAVE_POPSIFT)
-  INCLUDE_DIRECTORIES(${POPSIFT_INCLUDE_DIR})
-  LINK_DIRECTORIES(${POPSIFT_LIBRARY_DIR})
+  INCLUDE_DIRECTORIES(${PopSift_INCLUDE_DIR})
```

Comment:
this two are not necessary (include and link), when u do `target_link_directory(... PopSift::popsift)` this automagically imports the necessary includes and the libraries to link with.